torch/ao/quantization/qconfig.py

```diff
@@ -325,6 +329,11 @@ def get_default_qat_qconfig(backend='fbgemm', version=1):
                                                                 quant_min=0,
                                                                 quant_max=255),
                               weight=default_per_channel_weight_fake_quant)
+        if backend == 'x86':
```

Comment:
Can we specify the `reduce_range` argument explicitly?  It's specified above in L316:L325, but not specified here, so the reader would have to go look at the default to understand what's going on.  Ideally the `onednn` callsite would specify it too.

Reply:
Well. It was per @jerryzh168 's request. So, what is expected here after all?

Reply:
maybe we can have a follow up PR to remove `reduce_range` from the L316:L325?

Reply:
I think we should be consistent in the codebase with usage of this argument, either we use it everywhere or we deprecate it.  I don't think this PR is related to that decision, so might be better if it stayed consistent with the codebase today (which is using reduce_range).  We can consider deprecating it in a separate PR, to reduce confusion.

Reply:
I think we have tried to remove this before: https://github.com/pytorch/pytorch/pull/71027 but it's probably not exhaustive, so the current state for codebase is we use it in some places, so not sure being stay consistent with the codebase today means here

Reply:
sorry I looked again, looks like you are referring to being consistent with the other qconfigs in this function, I think we can add reduce_range here and then remove both in a follow up PR to be consistent

Reply:
Yes, that's correct.

Removing `reduce_range` is high risk as there have been a lot of issues in the past about people setting this flag incorrectly, and a lack of error checking in our codebase for its value.  I think to deprecate it we first need a documented plan with reasoning explained, just doing it in one-off PRs without a broader plan sounds too risky to me.

Reply:
I think the reason to remove reduce_range is not for deprecating this flag, but more of "new code should not be using this anymore". agree that will need more planning for deprecating (removing) this flag from the whole codebase

Reply:
IMO being consistent with the lines of code directly above this one should rank higher than "new code should not be using this anymore".

Reply:
yeah I'm talking about being consistent with the above code in this PR and remove both in next PR because "new code should not be using this anymore"

Reply:
Looks like you have reached an agreement. I will add `reduce_range` back. Thanks.

Reply:
> Looks like you have reached an agreement. I will add `reduce_range` back. Thanks.

@Xia-Weiwen Please be reminded that you have to add the following code back too.
https://github.com/pytorch/pytorch/pull/84329#discussion_r977174445

Reply:
Done

Reply:
OMG, nobody notices it is starting with `if` not `elif`, which is a huge mistake.

Reply:
Good catch! I guess the default path goes to `version == 1` so this issue was not observed earlier?

Reply:
Yes, was investigating https://github.com/pytorch/pytorch/issues/88800 so I set version to zero.